src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.module.scss

```diff
@@ -89,6 +88,18 @@ $laptop-range-small-end: 929px;
   z-index: $z-index-navbar;
 }
 
+.bgMoss {
+  background-color: var(--BrandMoss);
+  color: var(--BrandForest);
+}
```

Comment:
Would it be difficult to have this just be a transparent background, so that whatever module we place behind the navbar automatically sets the background color? 

It would also allow us to fade in the white background on scroll using the alpha channel.

Reply:
The color for nav bar is transparent by default and will only change to a different color when navbarColor is specified https://github.com/getethos/ethos-design-system/pull/492/files#diff-0218fe2c9ebf497571c90a9dc4cd522757d8d507b155bb70b1cabce02f3dad69R111
If the navbarColor is specified, then we will display the nav bar with the color passed by CMS. bgMoss class is the result of navbarColor='moss'.

Reply:
From what we talk today, I will remove the navbarColor since the color of the nav bar will depend on the single CTA's background color and the changes are being worked on in CMS.